tidb-cloud/tidb-cloud-performance-reference.md

```diff
@@ -21,7 +21,7 @@ Test environment:
 
 Test scale:
 
-- TiDB: 2 vCPU \* 2; TiKV: 2 vCPU \* 3
+- TiDB: 2 vCPU, 8 GiB \* 2; TiKV: 2 vCPU, 8 GiB \* 3
```

Comment:
It might misleading in the preview.
![image](https://user-images.githubusercontent.com/79440533/187398329-bbd0ea13-4288-4e47-976c-cd03cb5ebe15.png)

Reply:
```suggestion
- TiDB (2 vCPU, 8 GiB) \* 2; TiKV (2 vCPU, 8 GiB) \* 3
```

Reply:
```suggestion
- TiDB ( 2 vCPU, 8 GiB ) \* 2; TiKV ( 2 vCPU, 8 GiB ) \* 3
```